### ISSUE TITLE
pfe-accordion | Keyboard navigation not working

### ISSUE THREAD

**Issue by castastrophe** [patternfly/patternfly-elements#822]
https://www.redhat.com/en/search/RHEL

If you focus on a heading in the accordion, you can't tab or arrow key around to the other accordion headings :(

**Comment by kylebuch8** [comment]
@castastrophe Can you provide some more detail here? I only see one accordion header on the search page.

![accordion-search](https://user-images.githubusercontent.com/330256/78376967-060c3500-759d-11ea-9169-10e73e17dfd9.gif)

Use the tab key inside the accordion is doing what I'd expect. The tab order on the page feels a bit funky. When I'm done tabbing through the accordion, I'm sent to the pagination at the bottom of the page but that's expected after looking at the code. 

What am I missing?

**Comment by castastrophe** [comment]
I was looking at the accordions in the filter section but I realize now that they're custom code and not the web component!  😮 #mybad
